drivers/SmartThings/zigbee-switch/profiles/color-temp-bulb-2200K-4000K.yml

```diff
@@ -0,0 +1,20 @@
+name: color-temp-bulb-2200K-4000K
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: colorTemperature
+    version: 1
+    configuration:
```

Comment:
@ctowns `configuration` needs to be changed to `config` in all the profiles per @iplantemn

Reply:
Done